docs/tutorials/tutorial-jdbc.md

```diff
@@ -0,0 +1,129 @@
+- The `query` variable contains the SQL query you want to submit to Druid.
+
+## Get the metadata for a datasource
+
+Metadata, such as column names, is available either through the [`INFORMATION_SCHEMA`](../querying/sql-metadata-tables.md) table or through `connect.getMetaData()`. The following example uses the `INFORMATION_SCHEMA` table to retrieve and print the list of column names for the `wikipedia` datasource that you loaded during a previous tutorial.
+
+```java
+import java.sql.*;
+import java.util.Properties;
+
+public class JdbcListColumns {
+
+    public static void main(String args[]) throws SQLException
+    {
+        // Connect to /druid/v2/sql/avatica/ on your Broker.
+        String url = "jdbc:avatica:remote:url=http://localhost:8888/druid/v2/sql/avatica/";
+
+        String query = "SELECT COLUMN_NAME,* FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_NAME = 'wikipedia' and TABLE_SCHEMA='druid'";
+        // Set any connection context parameters you need here
+        // Or leave empty for default behavior.
+        Properties connectionProperties = new Properties();
+
+        try (Connection connection = DriverManager.getConnection(url, connectionProperties)) {
+            try (
+                    final Statement statement = connection.createStatement();
+                    final ResultSet rs = statement.executeQuery(query)
+            ) {
+                while (rs.next()) {
+                    String columnName = rs.getString("COLUMN_NAME");
+                    System.out.println(columnName);
+                }
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+
+    }
+}
+```
+
+## Query data
+
+Now that you know what columns are available, you can start querying the data. The following example queries the datasource named `wikipedia` for the timestamps and comments from Japan. It also sets two [query context parameters](../querying/sql-query-context.md), `sqlTimeZone` and `useCache`. In addition, you can also parameterize queries by using [dynamic parameters](../querying/sql-jdbc.md#dynamic-parameters).
+
+```java
+import java.sql.*;
+import java.util.Properties;
+
+public class JdbcCountryAndTime {
+
+    public static void main(String args[]) throws SQLException
+    {
+        // Connect to /druid/v2/sql/avatica/ on your Broker.
+        String url = "jdbc:avatica:remote:url=http://localhost:8888/druid/v2/sql/avatica/";
+
+        //The query you want to run.
+        String query = "SELECT __time, isRobot, countryName, comment FROM wikipedia WHERE countryName='Japan'";
+        // Set any connection context parameters you need here
+        // Or leave empty for default behavior.
+        Properties connectionProperties = new Properties();
+        connectionProperties.setProperty("sqlTimeZone", "America/Los_Angeles");
+        connectionProperties.setProperty("useCache", "false");
```

Comment:
suggest not using cache for the example. Time zone is a good enough example. You already link out to list of query context params.